### ISSUE TITLE
Bad example in docs for tf.multinomial

### ISSUE THREAD

**Issue by nikitakit** [tensorflow/tensorflow#2769]
I think this comment in the docstring for tf.multinomial is wrong:

```
samples = tf.multinomial([[1, -1, -1]], 10) # samples is equivalent to tf.zeros([1, 10], dtype=tf.int64)
```
Logits of [1, -1, -1] do not assign zero probability to class 1 and 2.  Additionally, multinomial op has no kernel for `logits` of integer type, which is what's happening here. (Note usage of 1, not 1.0)

Since I can't figure out what purpose this example serves, my suggestion would be to remove it entirely.

I'm using tensorflow 845fb7ab77fae53b849894925c7d13c1b8918675 with Python 3.

**Comment by concretevitamin** [comment]
Fixed by [f36659](https://github.com/tensorflow/tensorflow/commit/f36659f555027481788d14cc388cec1071a3182a).
